src/handlers/tests/status-retriever-handler.test.ts

```diff
@@ -0,0 +1,224 @@
+import type { APIGatewayEvent } from 'aws-lambda';
+import { ContextExamples } from '@aws-lambda-powertools/commons';
+import { handle } from '../status-retriever-handler';
+import logger from '../../commons/logger';
+import { marshall } from '@aws-sdk/util-dynamodb';
+import { DynamoDbService } from '../../services/dynamo-db-service';
+
+jest.mock('../../commons/logger.ts');
+jest.mock('../../services/dynamo-db-service');
+jest.mock('@smithy/node-http-handler');
+
+const testEvent: APIGatewayEvent = {
+  body: null,
```

Comment:
It seems that `yarn lint` is not a fan of null and wants undefined instead, however this type (APIGatewayEvent) uses a union type with null, should the linting rules be updated to accommodate this type or should another type be used here instead?